src/generate.ts

```diff
@@ -406,13 +407,18 @@ export async function generate(
         }
       }
     }
+    if (generateOptions.patchScript) {
+      const patchScript = require(generateOptions.patchScript)
+      await patchScript.patch(doc)
```

Comment:
This is moving forward in a good direction. Could you please 
```suggestion
      doc = await patchScript.patch(doc, swaggerParser)
```
so that
 - the patch script can return a new document value to use
 - the patch script can possibly use the same swagger parser as this tool